go.mod

```diff
@@ -2,7 +2,4 @@ module github.com/josharian/impl
 
 go 1.14
 
-require (
-	golang.org/x/tools v0.4.0
-	golang.org/x/xerrors v0.0.0-20220907171357-04be3eba64a2 // indirect
-)
+require golang.org/x/tools v0.4.0
```

Comment:
I think this is just the result of running `go mod tidy` on `main`. If you get something different, please shout?